libs/wingii/src/lib.rs

```diff
@@ -179,7 +248,15 @@ pub mod type_system {
 				return Ok(name.to_string());
 			}
 
-			let asm = self.load_assembly(&assembly_file)?;
+			// Get the module version, this is used for fingerprinting the JSII manifest
+			// needed for comparing it to the JSII manifest cache
+			let module_version = if let Some(Value::String(ver_str)) = package.get("version") {
+				Some(ver_str.clone())
+			} else {
+				None
+			};
```

Comment:
I think we're going to have an issue with a massive pile of `jsii.bincode` caches in the wingsdk folder in development.

I can't really think of a good way to do this other than disabling caching if the version of the jsii is `0.0.0`

Reply:
It also compares the modification date and file size of the .jsii in addition to the module version. So it should take care of cleaning the old ones when it encounters an update.

Reply:
also added a test for cleanups